search-api/src/main/java/edu/unc/lib/boxc/search/api/SearchFieldKey.java

```diff
@@ -64,6 +64,8 @@
     READ_GROUP("readGroup", "readGroup", "Read Group"),
     RESOURCE_TYPE("resourceType", "type", "Resource Type"),
     RESOURCE_TYPE_SORT("resourceTypeSort", "typeSort", "Resource Type Sort"),
+    RIGHT("right", "right", "Rights"),
```

Comment:
I don't think this currently matches the solr schema field definition, and I know its weird after our the last PR where I was saying to depluralize, but here and in the URI field I think it would be appropriate to use "rights"
```suggestion
    RIGHTS("rights", "rights", "Rights"),
```